src/services/CertificateGenerationService.ts

```diff
@@ -89,6 +89,7 @@ class CertificateGenerationService {
       hgv_fail: config.documentNames.vtg30,
       hgv_fail_bilingual: config.documentNames.vtg30_bilingual,
       hgv_prs: config.documentNames.hgv_prs,
+        hgv_prs_bilingual: config.documentNames.hgv_prs_bilingual,
```

Comment:
something in the rebasing/linting has nudged this out of line

Reply:
I'll push up the change now.